packages/toolkit/src/createAsyncThunk.ts

```diff
@@ -542,6 +552,18 @@ export const createAsyncThunk = /* @__PURE__ */ (() => {
         }),
       )
 
+    function getError(x: unknown): GetSerializedErrorType<ThunkApiConfig> {
+      if (options && options.serializeError) {
+        return options.serializeError(x)
+      }
+
+      if (creatorOptions && creatorOptions.serializeError) {
+        return creatorOptions.serializeError(x, miniSerializeError)
```

Comment:
because we export miniSerializeError, i don't think we need to pass it here - the user can just import it if they want it

Reply:
i'd also prefer something like ```ts
const serializeError = options?.serializeError || creatorOptions?.serializeError || miniSerializeError
``` rather than this long function